--- flaky-3.6.1/test/test_nose/test_flaky_nose_plugin.py.orig	2019-01-09 18:29:18.000000000 +0100
+++ flaky-3.6.1/test/test_nose/test_flaky_nose_plugin.py	2020-01-13 16:06:51.602083798 +0100
@@ -5,8 +5,12 @@
 from unittest import TestCase
 
 from genty import genty, genty_dataset
-import mock
-from mock import MagicMock, Mock, patch
+try:
+    import mock
+    from mock import MagicMock, Mock, patch
+except ImportError:
+    from unittest import mock
+    from unittest.mock import MagicMock, Mock, patch
 
 from flaky import defaults, flaky_nose_plugin
 from flaky.flaky_decorator import flaky
--- flaky-3.6.1/test/test_pytest/test_flaky_pytest_plugin.py.orig	2019-08-06 20:25:51.000000000 +0200
+++ flaky-3.6.1/test/test_pytest/test_flaky_pytest_plugin.py	2020-01-13 16:07:18.311939098 +0100
@@ -2,7 +2,11 @@
 
 from __future__ import unicode_literals
 from io import StringIO
-from mock import Mock, patch
+try:
+    from mock import Mock, patch
+except ImportError:
+    from unittest.mock import Mock, patch
+
 # pylint:disable=import-error
 import pytest
 from _pytest.runner import CallInfo
